src/plugins/intel_gpu/src/graph/impls/cpu/strided_slice.cpp

```diff
@@ -85,6 +85,10 @@ struct strided_slice_impl : public typed_primitive_impl<strided_slice> {
         OV_ITT_SCOPED_TASK(ov::intel_gpu::itt::domains::intel_gpu_plugin, "strided_slice::execute_impl");
         auto& stream = instance.get_network().get_stream();
 
+        if (instance.can_be_optimized()) {
+            return aggregate_events(events, stream, false, instance.is_output());
```

Comment:
Hmm, wouldn't it introduce extra synchronizations with GPU Kernels? I suppose, instance.is_output() will be always false, and events list is not empty, right? In that case for OOO queue we may reuse some existing previously enqueued barrier (during enqueue marker call)  and introduce extra synchronization with GPU kernels for ShapeOf subgraphs impls.
Did you have a chance to try events grouping?
```suggestion
            return aggregate_events(events, stream, true, instance.is_output());
```

Reply:
oh yes, I didn't overlooked the parameters. Let me fix more. Thanks!

Reply:
I fixed, thanks